docs/detections/alerts-view-details.asciidoc

```diff
@@ -26,7 +26,11 @@ The *Overview* tab contains these features:
 
 * *Summary*: Displays general details such as the alert's status, severity, risk score, and a link to the detection rule that produced the alert.
 
-* *Reason statement*: Provides a description of what generated the alert and provides general alert details. You can use this to understand the alert's origin and determine if the alert is relevant to your investigation.
+* *Alert rendered reason statement*: Summarizes the alert and renders the statement in a human-readable format. High-level alert details are provided, including the alert severity (`kibana.alert.severity`) and the rule that generated the alert (`kibana.alert.rule.name`). Alert details are interactive. Hover over them to access multiple actions.
```

Comment:
Consider removing the text  ~and renders the statement~, shown as ~strikethrough~ below:

*Alert rendered reason statement*: Summarizes the alert ~and renders the statement~ in a human-readable format. High-level alert details are provided, including the alert severity (`kibana.alert.severity`) and the rule that generated the alert (`kibana.alert.rule.name`). Alert details are interactive. Hover over ~them~ to access multiple actions.

Also consider (in the text above) replacing ~them~ with _fields_, such that the new text reads:

Hover over ~them~ fields to access multiple actions.